core/controllers/base.py

```diff
@@ -425,7 +459,13 @@ def _render_exception(self, error_code, values):
                 self.render_template('pages/error/error.html')
 
     def handle_exception(self, exception, unused_debug_mode):
-        """Overwrites the default exception handler."""
+        """Overwrites the default exception handler
+
+        Args:
+            exception: int. Raised while assertion of error code
```

Comment:
exception is not an int. Why do you think it is? Try to generate an exception in the dev server, log its type, and compare with what you have here.